### ISSUE TITLE
Unable to reproduce the results in your paper

### ISSUE THREAD

**Issue by v-wewei** [Improbable-AI/dexenv#2]
Dear authors:

I am using the released code for reproduce the teacher network for in-hand object rotation in simulator, while i am not able to achieve the results list in the paper. In fact , the model just learn to grasp the obejct to prevent the object to fall down, and the rotation reward of the policy have a mean value around 0.6 after training about 4 days. I have not modified anything.  i found the released pretrained teacher model preform well in simulation. Could you please help to check if the training hyperparams for the teacher policy. Thanks advance.

**Comment by sjtu-cjj** [comment]
Hello, I am also working on reproducing this paper recently and have encountered the same issue with poor training results. May I ask what GPU you used for the four-day training?

**Comment by v-wewei** [comment]
> Hello, I am also working on reproducing this paper recently and have encountered the same issue with poor training results. May I ask what GPU you used for the four-day training?

An RTX 3060 GPU

**Comment by kaixuanwang2003** [comment]
Same here, the provided pretrained checkpoint for the student policy seems not working, objects keep falling down
